src/main/webapp/app/exercises/text/assess-new/text-submission-assessment.route.ts

```diff
@@ -12,39 +12,44 @@ import { TextFeedbackConflictsComponent } from './conflicts/text-feedback-confli
 import { Authority } from 'app/shared/constants/authority.constants';
 
 @Injectable({ providedIn: 'root' })
-export class StudentParticipationResolver implements Resolve<StudentParticipation | undefined> {
-    constructor(private textAssessmentsService: TextAssessmentsService) {}
+export class NewStudentParticipationResolver implements Resolve<StudentParticipation | undefined> {
+    constructor(private textSubmissionService: TextSubmissionService) {}
 
     /**
      * Resolves the needed StudentParticipations for the TextSubmissionAssessmentComponent using the TextAssessmentsService.
      * @param route
      */
     resolve(route: ActivatedRouteSnapshot) {
-        const submissionId = Number(route.paramMap.get('submissionId'));
-
-        if (submissionId) {
-            return this.textAssessmentsService.getFeedbackDataForExerciseSubmission(submissionId).catch(() => Observable.of(undefined));
+        const exerciseId = Number(route.paramMap.get('exerciseId'));
+        console.log('router: resolve:');
+        const correctionRound = Number(route.queryParamMap.get('correction-round'));
+        console.log('correctionRound: ', correctionRound);
```

Comment:
Same here.

Reply:
removed them here too. thanks for noticing :)